src/components/camera-tool.js

```diff
@@ -88,6 +88,12 @@ AFRAME.registerComponent("camera-tool", {
       selfieScreen.scale.set(-2, 2, 2);
       this.el.setObject3D("selfieScreen", selfieScreen);
 
+      this.cameraSystem = this.el.sceneEl.systems.cameras;
+
+      if (this.cameraSystem) {
```

Comment:
I would ditch this defensive if, if the system doesn't exist something bad has happened, and this if would just put us in a weird state where this camera inst registered instead of complaining loudly.